service-front/module/Application/view/application/pages/abandoned_flow.twig

```diff
@@ -0,0 +1,75 @@
+{% extends "layout/layout" %}
+{% block content %}
+    {% include 'layout/id_check_banner.twig' with details_data %}
+    <form method="post" class="govuk-!-width-two-thirds">
+        <div class="govuk-width-container govuk-!-padding-top-5">
+            <h1 class="govuk-heading-xl">
+                {% block title %}
+                    Exit identity check?
+                {% endblock %}
+            </h1>
+            <div class="govuk-form-group">
+                <input type="hidden"  name="last_page" value="{{ last_page }}">
+                <div class="govuk-form-group {% if form.get('reason').messages %}govuk-form-group--error{% endif %}">
+                <fieldset class="govuk-fieldset">
+                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                        <h2 class="govuk-fieldset__heading govuk-heading-m">
+                            Reason
+                        </h2>
+                    </legend>
+                    {% if form.get('reason').messages %}
+                        <p id="dln-error" class="govuk-error-message">
+                            <span class="govuk-visually-hidden">Error:</span>
+                            {{ form.get('reason').messages | join(', ', ' and ') }}
+                        </p>
+                    {% endif %}
+                    <div class="govuk-radios" data-module="govuk-radios">
+                        <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" id="reason" name="reason" type="radio" value="cd">
```

Comment:
We should probably figure out what the business wants to happen with this information, at the moment it's just being ignored